docs/DeveloperGuide.md

```diff
@@ -168,10 +168,10 @@ have been omitted from the diagram. Those details are shown in a separate sequen
     presence of the variable in user input.
  
  
-- Alternative 2: Require all values of an <code>item</code> object to be updated
-  - Pros: Will have less code to deal with having no additional parsing of input string needed.
+- Alternative 2: Require all values of an <code>item</code> object to be updated.
+  - Pros: Will have less code to deal with having no additional parsing of the input string.
   
-  - Cons: Less user flexibility; user must input all parameters even if he/she does not want to update certain
+  - Cons: Less user flexibility; user must input all parameters even if he/she does not wish to update certain
   variables.
 
```

Comment:
Maybe can add a statement to explain why we choose alternative 1 over alternative 2 as mentioned in the DG review.

Reply:
okie thank you will do it!